.github/workflows/ci-app-tests.yml

```diff
@@ -101,6 +101,12 @@ jobs:
         if: ${{ matrix.pkg-name == 'LAI' }}
         run: python .actions/assistant.py copy_replace_imports --source_dir="./tests" --source_import="lightning_app" --target_import="lightning.app"
 
+      - name: Adjust examples
```

Comment:
hmm, why won't we just change the examples to whatever a user must be doing?

Reply:
Not sure if I follow...
the case is that in general, we want to showcase `lightning` as a monolithic package, but on the other hand, we still allow install `lighntning_app` as a stand-alone package, and in such case, these examples are not valid...
Why change it only in CI? because some are copied in pytest runtime, and I found it much more readable to edit all examples that over-engineer the pytest function
https://github.com/Lightning-AI/lightning/blob/59c65db81c2bcf22477fff89b55d1a511a9cc660/tests/tests_app/conftest.py#L23-L36

Reply:
Yeah, it's like, if LAI is installed, examples use `import lightning`. but if lightning_app is installed, examples use `import lightning_app`